core/modules/update/update.theme.inc

```diff
@@ -42,8 +42,9 @@ function theme_update_last_check($variables) {
   $output = '<div class="update checked">';
 
   $output .= $last ? t('Last checked: @time ago', array('@time' => format_interval(REQUEST_TIME - $last))) : t('Last checked: never') . ' ';
-  $output .= l(t('Check manually'), 'admin/reports/updates/check', array('attributes' => array('class' => array('button', 'button-secondary')), 'query' => backdrop_get_destination()));
-
+  if (user_access('access site reports')) {
```

Comment:
It turns out that this is the wrong permission to check. We should be checking the permission for: `Administer site configuration`

That is the permission that controls if the user is able to manually check for updates.

Reply:
Done